host-bin/mount-chroot

```diff
@@ -216,32 +227,81 @@ $passphrase" | ecryptfs-wrap-passphrase "$wrappedfnek" -
             tail -c 160 "$KEYFILE" | head -c 80 > "$wrappedkey"
             tail -c 80 "$KEYFILE" > "$wrappedfnek"
 
+            PAD1="`cat "$wrappedkey" | head -c 16`"
```

Comment:
echo, not cat.  But you can also do:

`if [ "${wrappedkey#................}" != "$PAD1" -a "${wrappedfnek#................}" != "$wrappedfnek" ]; then`